collector/meminfo_darwin.go

```diff
@@ -59,5 +68,8 @@ func (c *meminfoCollector) getMemInfo() (map[string]float64, error) {
 		"swapped_in_bytes_total":  ps * float64(vmstat.pageins),
 		"swapped_out_bytes_total": ps * float64(vmstat.pageouts),
 		"total_bytes":             float64(total),
+		"swap_used_bytes":         float64(swap.xsu_used),
+		"swap_free_bytes":         float64(swap.xsu_avail),
```

Comment:
It looks like Darwin has correct accounting, so "used + free == total". Because of this, we can skip having a "free" metric and derive free. IE "total - used = free".